save-points/00-Starting-point/BlazingPizza.Server/SpecialsController.cs

```diff
@@ -20,7 +20,7 @@ public SpecialsController(PizzaStoreContext db)
         [HttpGet]
         public async Task<ActionResult<List<PizzaSpecial>>> GetSpecials()
         {
-            return await _db.Specials.OrderByDescending(s => s.BasePrice).ToListAsync();
+            return (await _db.Specials.ToListAsync()).OrderByDescending(s => s.BasePrice).ToList();
```

Comment:
This is a sadpants limitation. I think it's probably due to sqlite + how we mapped this column. I'm ok with the workaround since this isn't really the point of the workshop.